commands/command_divelist.cpp

```diff
@@ -469,10 +469,10 @@ ImportDives::ImportDives(struct dive_table *dives, struct trip_table *trips, str
 	// this only matters if undoit were called before redoit
 	currentDive = nullptr;
 
-	struct dive_table dives_to_add = { 0 };
-	struct dive_table dives_to_remove = { 0 };
-	struct trip_table trips_to_add = { 0 };
-	struct dive_site_table sites_to_add = { 0 };
+	struct dive_table dives_to_add = { 0, 0, 0 };
```

Comment:
This is good - we had that `{ 0 }` construct in some places and I copied it.

Perhaps we should add `empty_*_table` objects so that we can write
`struct dive_table dives_to_add = empty_dive_table`

IIRC it was completely obvious to make this work with C and C++ at the same time...

Reply:
Updated previous comment - should of course read "not completely obvious".